### PR TITLE
Prefer using "call" to "invoke" applied to methods

### PR DESCRIPTION
**Pull request by pirj**
I believe "invoke" is rarely used in the Ruby landscape, and "call" is used more often.

[`Method` has a `call` method](https://ruby-doc.org/core-3.0.0/Method.html#method-i-call).
[`Proc` has a `call` method](https://ruby-doc.org/core-3.0.0/Proc.html#method-i-call).

Also,

> As verbs the difference between call and invoke is that call is (lb) to use one's voice while invoke is to call upon (a person, especially a god) for help, assistance or guidance.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -4969,7 +4969,7 @@ Use `%r` only for regular expressions matching _at least_ one `/` character.
 
 === `%x` [[percent-x]]
 
-Avoid the use of `%x` unless you're going to invoke a command with backquotes in it (which is rather unlikely).
+Avoid the use of `%x` unless you're going to execute a command with backquotes in it (which is rather unlikely).
```
Comment (pirj) [inline README.adoc:4972]: https://ruby-doc.org/core-2.7.1/Kernel.html#method-i-system

> Executes command

File: README.adoc [hunk 2]
```diff
@@ -2723,9 +2723,9 @@ def some_method_with_parameters(param1, param2)
 end
 ----
 
-=== Method Invocation Parentheses [[method-invocation-parens]]
+=== Method Call Parentheses [[method-invocation-parens]][[method-call-parens]]
```
Comment (pirj) [inline README.adoc:2726]: Kept old anchors.

AsciiDoc reference for additional anchors https://docs.asciidoctor.org/asciidoc/latest/sections/custom-ids/#assign-additional-ids

**Comment by bbatsov** [comment]
Fine by me. I'm reasonably sure the usage of invocation came from me, as I've alwasy been fond of the term, but I'm certainly not attached to it. :-)

**Comment by bbatsov** [comment]
> As verbs the difference between call and invoke is that call is (lb) to use one's voice while invoke is to call upon (a person, especially a god) for help, assistance or guidance.

Btw, this actually confirms that "invoke" is the better term in the context of programming. :D

**Comment by bbatsov** [comment]
Here there's a nice conversation on the subject - https://stackoverflow.com/questions/50884893/calling-vs-invoking-a-function

**Pull request merged**